### PR TITLE
Makes the IC issue message a little bit less 'you ded lol'

### PR DESCRIPTION
**Pull request by adamsong**
# Github documenting your Pull Request
Improved the IC issue button response.

# Changelog

:cl:  
tweak: tweaked the 'Marked as IC' message to be a bit kinder
/:cl:

### CODE REVIEW

File: yogstation/code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -381,8 +381,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 		return
 
 	var/msg = "<font color='red' size='4'><b>- AdminHelp marked as an IC issue by [usr.client.holder?.fakekey ? "an Administrator" : key_name(usr, 0, 0)]! -</b></font><br>"
-	msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
-	msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font>"
+	msg += "<font color='red'>Unfortunately, Your issue does not warrent admin intervention, usually becuase none of the involved parties are breaking any OOC rules. You can still attmept to find an IC solution to your problem however.</font>"
```
Comment (Bibby0110) [inline yogstation/code/modules/admin/verbs/adminhelp.dm]:
```suggestion
	msg += "<font color='red'>Unfortunately your issue does not warrant admin intervention, usually because none of the involved parties are breaking any OOC rules. You can however still attempt to find an IC solution to your problem.</font>"
```

**Comment by Xoxeyos** [comment]
You can just shorten this to "Your issue does not warrant admin intervention, you have the ability to solve this IC."; The issue with "...usually because none of the involved parties are breaking any OOC rules." is that OOC rules are not the only rules (obviously), there's just a better way to word this stuff.

Here may be a better way to word it; "Your issue doesn't warrant admin intervention, you have the ability on solving this in character." And maybe add in something like this, "If you believe a user is in the wrong, you may make a complaint against them.".

Also, I believe "Dying is part of the game." should stick in somewhere.

**Review by JamieD1 — APPROVED** [review]
Finally
